modules/nw-egress-service-ovn.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services.adoc
+
+:_content-type: PROCEDURE
+[id="nw-egress-service-ovn_{context}"]
+= Deploying an egress service
+
+You can deploy an egress service to manage egress traffic for pods behind a `LoadBalancer` service.
+
+The following example configures the egress traffic to have the same source IP address as the ingress IP address of the `LoadBalancer` service.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* You configured MetalLB `BGPPeer` resources.
+// Can we make this part of the procedure?
+
+.Procedure
+
+. Create an `IPAddressPool` custom resource (CR) with the desired IP for the service. 
+
+.. Create a file, such as `ip-addr-pool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: example-pool
+  namespace: metallb-system
+spec:
+  addresses:
+  - 172.19.0.100/32
+  autoAssign: false <1>
+----
+<1> Set `autoAssign` to `false` to ensure MetaLB reserves IP addresses in the IP address pool exclusively for the `LoadBalancer` service.
+
+.. Apply the configuration for the IP address pool by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f ip-addr-pool.yaml
+----
+
+. Create `Service` and `EgressService` CRs. 
+
+.. Create a file, such as `service-egress-service.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: example-namespace
+  annotations:
+    metallb.universe.tf/address-pool: example-pool <1>
+spec:
+  selector:
+    app: example
+  ports:
+    - name: http
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
+  type: LoadBalancer
+---
+apiVersion: k8s.ovn.org/v1
+kind: EgressService
+metadata:
+  name: example-service
+  namespace: example-namespace
+spec:
+  sourceIPBy: "LoadBalancerIP" <2>
```

Comment:
same considerations about sourceIP / node selector I made in other places

Reply:
At the top of this procedure, I say `The following example configures the egress traffic to have the same source IP address as the ingress IP address of the `LoadBalancer` service.` So perhaps it's ok in this example. 
The `sourceIPBy` field is essentially required so I think it must be doc'd with either`network` or `LoadBalancer`. 
I add "This example uses the `LoadBalancerIP` value..." in the callout to show it's just an example. The [content detailing possible EgressService CR specifications](https://github.com/openshift/openshift-docs/pull/61548/files#diff-21d7502acf2deac8ff633b37a9acc056d3fd8b22805e3729f587ccf69c1a372fR26-R27) hopefully makes it clear that `nodeSelector` is optional and only when you set the `LoadBlanacer` as the source IP